### ISSUE TITLE
Delete model from Explorer does not delete related representations

### ISSUE THREAD

**Issue by arnauddieumegard** [eclipse-syson/syson#425]
![delete model not representation](https://github.com/eclipse-syson/syson/assets/1576016/4a1bdafa-3ec9-4869-8f77-95080895066d)

This ends with:
```
Exception while fetching data (/diagramEvent) : Cannot invoke "org.eclipse.sirius.components.representations.IRepresentation.getId()" because "representation" is null
```

**Comment by AxelRICHARD** [comment]
Probably the problem raised in Sirius Web : https://github.com/eclipse-sirius/sirius-web/issues/3630

**Comment by AxelRICHARD** [comment]
Fixed with Sirius Web 2024.5.3
